docs/integrations/databases/neon.mdx

```diff
@@ -119,72 +104,100 @@ export const db = drizzle(sql, {
 ```
 </CodeBlockTabs>
 
-### Generate and run database migrations
+### Push the schema to the database
 
-Use the `drizzle-kit` package to generate and run database migrations.
+To load the schema into the database, create a `drizzle.config.ts` file at the root of your project and add the following configuration:
 
-1. At the root of your project, create a `drizzle.config.ts` and add the following configuration:
-    ```typescript {{ filename: 'drizzle.config.ts' }}
-    import { defineConfig } from "drizzle-kit";
-    import { loadEnvConfig } from "@next/env";
+```typescript {{ filename: 'drizzle.config.ts' }}
+import { defineConfig } from "drizzle-kit";
+import { loadEnvConfig } from "@next/env";
 
-    loadEnvConfig(process.cwd());
+loadEnvConfig(process.cwd());
 
-    if (!process.env.DATABASE_URL) {
-      throw new Error("DATABASE_URL must be a Neon postgres connection string");
-    }
+if (!process.env.DATABASE_URL) {
+  throw new Error("DATABASE_URL must be a Neon postgres connection string");
+}
 
-    export default defineConfig({
-      dialect: "postgresql",
-      dbCredentials: {
-        url: process.env.DATABASE_URL,
-      },
-      schema: "./app/db/schema.ts",
-      out: "./migrations",
-    });
-    ```
-1. Generate and run the database migrations by running the following commands:
-    ```sh {{ filename: 'terminal' }}
-    npx drizzle-kit generate
-    npx drizzle-kit migrate
-    ```
+export default defineConfig({
+  dialect: "postgresql",
+  dbCredentials: {
+    url: process.env.DATABASE_URL,
+  },
+  schema: "./app/db/schema.ts",
+});
+```
+
+Run the following command to push the schema to the database:
+```sh {{ filename: 'terminal' }}
+npx drizzle-kit push
+```
 
-### Create the UI of the home page
+### Creating server actions with Clerk
 
-Add the following code to the `app/page.tsx` file to create the UI of the home page:
+To add and delete user messages from the database, create an `actions.ts` file in the `app` directory. Use the `currentUser` function from the Clerk SDK to get the user ID and interact with the database.
```

Comment:
```suggestion
Create an `actions.ts` file in your `app` directory and add the following code in order to create Server Actions that will handle your form submissions. You will have two forms: one for creating and one for deleting messages.

To interact with the database, Clerk's [`currentUser()`](/docs/references/nextjs/current-user) helper is used to get ID of the current user of your application. Then, the ID is either attached to a newly created message or used to find an existing message for deletion.
```

Reply:
I think the functions are easy enough to read, so we can skip the last sentence.

```suggestion
To handle form submissions for adding and deleting user messages, create two Server Actions in `app/actions.ts`. Use the `auth()` function from Clerk to obtain the user ID, which will be used to interact with the database.
```